app/src/androidTest/java/org/openobservatory/ooniprobe/client/OONIAPIClientTest.java

```diff
@@ -66,4 +70,51 @@ public void onError(String msg) {
     @Test
     public void testDeleteJsons() {
     }
+
+    @Test
+    public void getMeasurementJsonSuccess() {
+        final CountDownLatch signal = new CountDownLatch(1);
+        new OkHttpClient().newCall(new Request.Builder().url(JSON_URL).build()).enqueue(new GetMeasurementCallback() {
+            @Override
+            public void onSuccess(String json) {
+                Assert.assertNotNull(json);
+                signal.countDown();
+            }
+
+            @Override
+            public void onError(String msg) {
+                Assert.fail();
+                signal.countDown();
+            }
+        });
+        try {
+            signal.await(1, TimeUnit.MINUTES);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+            Assert.fail();
+        }
+    }
+
+    @Test
+    public void getMeasurementJsonError() {
+        final CountDownLatch signal = new CountDownLatch(1);
+        new OkHttpClient().newCall(new Request.Builder().url(NON_PARSABLE_URL).build()).enqueue(new GetMeasurementCallback() {
```

Comment:
Here instead of a non parsable URL you should pass a nonexisting URL. This should probably fix the CI.

`https://ooni.explorer.io/nonexistent-measurement-url`

Reply:
Also here use the singleton